server/prisma/schema.prisma

```diff
@@ -193,7 +193,6 @@ model User {
   id            String          @default(cuid()) @id
   email         String          @unique
   name          String
-  password      String
```

Comment:
we will need to migrate all users to identity-provider when applying this migration where the stack is already deployed, will we do this manually ?

Reply:
I think we'll need a script to do that